### PR TITLE
Improved opening sentence (I hope)

### PR DESCRIPTION
**Pull request by udidahan**

### CODE REVIEW

File: transports/msmq/routing.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ redirects:
  - nservicebus/msmq/routing
 ---
 
-MSMQ is a [bus transport](/transports). This means that an MSMQ system consists of multiple nodes, one on each machine, forwarding messages between each other. Each endpoint connects to its local MSMQ node. In order to address a different endpoint, not only the queue name but also the host name has to be specified.
+The MSMQ transport in NServiceBus is a distributed transport in which there is the [MSMQ](https://msdn.microsoft.com/en-us/library/ms711472(v=vs.85).aspx) queue process running on each machine, with messages being stored locally before being forwarded to other machines. Each NServiceBus endpoint connects to its local MSMQ node. In order to address a different endpoint, not only the queue name but also the host name has to be specified.
```
Comment (SimonCropp) [inline transports/msmq/routing.md]: "in NServiceBus" would seem redundant in this context

Comment (SimonCropp) [inline transports/msmq/routing.md]: "NServiceBus endpoint" in other doco the "NServiceBus " is omitted and just "endpoint" used

Comment (udidahan) [inline transports/msmq/routing.md]: I wanted to clearly distinguish between MSMQ itself and our transport on top of it.

I do agree about leaving just "endpoint" - though I'll quote it.

**Comment by udidahan** [comment]
Hopefully this latest commit helps.

File: transports/msmq/routing.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ redirects:
  - nservicebus/msmq/routing
 ---
 
-The MSMQ transport in NServiceBus is a distributed transport in which there is the [MSMQ process](https://msdn.microsoft.com/en-us/library/ms711472(v=vs.85).aspx) running on each machine, with messages being stored locally before being forwarded to other machines. In this model, each `endpoint` connects to the local MSMQ process and, when addressing a different endpoint, not only the queue name but also the host name has to be specified.
+The MSMQ transport in NServiceBus is a distributed transport in which there is the [MSMQ process](https://msdn.microsoft.com/en-us/library/ms711472.aspx) running on each machine, with messages being stored locally before being forwarded to other machines. In this model, each `endpoint` connects to the local MSMQ process and, when addressing a different endpoint, not only the queue name but also the host name has to be specified.
```
Comment (SimonCropp) [inline transports/msmq/routing.md]: @udidahan just a note one this commit i did. we usually link to the new version of doco, and for msdn that means omitting the `(v=vs.85)`

**Pull request merged**